src/matomo-settings.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Matomo settings for tracking statistics. This file could be mounted in the docker container.
+ */
+export const matomoSettings = {
+  hostUrl: 'http://localhost:8135/',
```

Comment:
?

Reply:
It is example URL, which must be changed. I could be changed via volume in the docker container as I described above.